DDFIngestTest.py

```diff
@@ -0,0 +1,66 @@
+import sys, requests, re, time
+
+def main():
+    if len(sys.argv) != 4:
+        print_usage_statement()
+        return
+
+    ingest_type = sys.argv[1]
+    file_path = sys.argv[2]
+    xml_type = sys.argv[3]
+    file = open(file_path, "r")
+    file_contents = file.read()
```

Comment:
Python with statement (context manager) https://docs.python.org/3.6/reference/compound_stmts.html#with
(like a try-with-resources in java). 
```python
file_contents = None
with open(file_path, "r") as file:
    file_contents = file.read()
```